src/main/java/net/mcreator/element/converter/ConverterRegistry.java

```diff
@@ -79,8 +79,7 @@ public class ConverterRegistry {
 				new ProcedureVariablesConverter(), new ProcedureVariablesEntityFixer(),
 				new LegacyProcedureBlockRemover(), new LegacyBlockPosProcedureRemover(), new ProcedureShootArrowFixer(),
 				new LegacyShootArrowProcedureRemover(), new BiomeDictionaryProcedureConverter(),
-				new ToolToItemTypeProcedureConverter()));
-		put(ModElementType.RANGEDITEM, List.of(new RangedItemTextureConverter()));
+				new ToolToItemTypeProcedureConverter(), new ProcedureArrowProjectileFixer()));

```

Comment:
```suggestion
				new ToolToItemTypeProcedureConverter(), new ExplodeProcedureConverter(),
				new ProcedureArrowProjectileFixer()));
```

Reply:
I will do it when merging master to avoid any problem

Reply:
No problem, I just showed the spot of the very conflict.